docs/contributor-guide/developer-guide/releasing.md

```diff
@@ -26,13 +26,27 @@ On the running servers inspect the DOM tree for `<head><meta name="civiform-buil
 * [staging-AWS](https://staging-aws.civiform.dev/)
 * [Seattle staging](https://staging.seattle.civiform.com/)
 
-### 2. Run the script to create a new draft release
+### 2. Create a New Draft Release
 
-You'll need to determine:
+Before proceding you'll need to determine:
 
 * Commit SHA: This is the middle item in the SNAPSHOT-sha-hash ID from Step 1
 * Release number: Pick the next minor number (1.X.0) since the [last release](https://github.com/civiform/civiform/releases)
 
+With those values create the draft release with one of the following options.
+
+#### Option 1: Via GitHub Actions
+
+* [Step 1 & 2] Navigate to the [Create Release](https://github.com/civiform/civiform/actions/workflows/release.yaml) GitHub Action.
```

Comment:
Why does it say "Step 1 & 2"? When reading instructions is looks just like 1 step.

Reply:
Oh it's because it's a direct link to the create release action, but in the image 1 & 2 are if you are clicking in from the action tab. I can reword that.